### DISCUSSION TITLE
Hourly rate excluding VAT??

### DISCUSSION THREAD

**Discussion by LordBritish12** [kimai/kimai#3544] [Questions & Answers] [answered]
Hi,

I would like to have my hourly rate exclude VAT. So e.g. charging 40€ per hour results in a bill for 1 hour of 47,6€ (given 19% VAT).
Is that possible? If so how do I configure that?

Thanks for your help,

Stefan

**Comment by kevinpapst** [comment, marked as answer]
Welcome Stefan 👋 
Your hourly rate is always without tax.
Your invoicing system calculates the VAT for your bill.
In Kimai the VAT rate is set on the invoice template.
Cheers
Kevin

**Comment by LordBritish12** [comment]
Oh, 
my bad, tested it with 50min. instead of an hour and got mixed up. 
Definetely works as required, Thanks for your swift reply.
Cheers,
Stefan
